packages/brick-kit/src/BrickWrapper.tsx

```diff
@@ -22,7 +22,7 @@ interface BrickWrapperProps {
 export function BrickWrapper(props: BrickWrapperProps): React.ReactElement {
   const locale =
     i18n.language && i18n.language.split("-")[0] === "en" ? enUS : zhCN;
-  const featureFlags = getRuntime().getFeatureFlags();
+  const featureFlags = getRuntime()?.getFeatureFlags?.() || {};
```

Comment:
这样会让生产环境每个构件渲染都会多几次没有意义的判断。试试这样：

```suggestion
  const featureFlags = process.env.NODE_ENV === "test" ? {} : getRuntime().getFeatureFlags();
```

这样在生产环境消费的 webpack 打包的制品，会在产出中剔除 `{}` 的分支，而在测试中使用的是 rollup 打包的代码，会保留并走到 `{}` 的逻辑。

Reply:
已改